### ISSUE TITLE
Get list of CKAN Routes

### ISSUE THREAD

**Issue by nickumia-reisys** [GSA/data.gov#4042]
## Purpose

We want to have block non-CKAN routes, but we're not sure how to do that.

Given above question, conducting [research/design/investigation/prototyping] is needed to provide factual knowledge on future steps.  We would like to have documentation about the finite set of CKAN urls that we support.

1 day of effort has been allocated and once compete, findings will be demonstrated and specific future actions will be decided.

### Acceptance Criteria

[ACs should be clearly demo-able/verifiable whenever possible. Try specifying them using [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications).]

- [x] GIVEN I look at this issue \
  WHEN 1 dayexpires\
  THEN I see a list of CKAN routes per app \
  AND [future action is decided] \
  AND [stories covering future action are created if needed]

### Background

[Any helpful contextual notes or links to artifacts/evidence, if needed]
To support
- https://github.com/GSA/data.gov/issues/3635

### Sketch

[Notes or a checklist reflecting our plan for tackling the uncertainty]

**Comment by nickumia-reisys** [comment]
<details>
  <summary>Catalog routes</summary>

   ```python
   ('/qa/link_checker', 'qa_blueprint.qa_resource_checklink')
   ('/report', 'report.index')
   ('/report/<report_name>', 'report.view')
   ('/report/<report_name>/<organization>', 'report.view')
   ('/dcat-us/validator', 'datajsonvalidator.validator')
   ('/', 'datagovtheme.redirect_homepage')
   ('/harvest/delete/<id>', 'harvester.delete')
   ('/harvest/refresh/<id>', 'harvester.refresh')
   ('/harvest/admin/<id>', 'harvester.admin')
   ('/harvest/about/<id>', 'harvester.about')
   ('/harvest/clear/<id>', 'harvester.clear')
   ('/harvest/<source>/job', 'harvester.job_list')
   ('/harvest/<source>/job/last', 'harvester.job_show_last')
   ('/harvest/<source>/job/<id>', 'harvester.job_show')
   ('/harvest/<source>/job/<id>/abort', 'harvester.job_abort')
   ('/harvest/object/<id>', 'harvester.object_show')
   ('/dataset/harvest_object/<id>', 'harvester.object_show')
   ('/usasearch-custom-feed.xml', 'geodatagov.feed')
   ('/topics-csv/{date}', 'geodatagov.csv')
   ('/topics-csv', 'geodatagov.csv')
   ('/api/2/search/<register>/geo', 'spatial_api.spatial_query')
   ('/api/2/rest/harvestobject/<id>/xml', 'spatial_harvest_metadata.harvest_object_redirect_xml')
   ('/api/2/rest/harvestobject/<id>/html', 'spatial_harvest_metadata.harvest_object_redirect_html')
   ('/harvest/object/<id>/original', 'spatial_harvest_metadata.display_xml_original')
   ('/harvest/object/<id>/html', 'spatial_harvest_metadata.display_html')
   ('/harvest/object/<id>/html/original', 'spatial_harvest_metadata.display_html_original')
   ('/catalog.<_format>', 'dcat.read_catalog')
   ('/dataset/<_id>.<_format>', 'dcat.read_dataset')
   ('/dcat.json', 'dcat_json_interface.dcat_json')
   ('/acs', 'saml2auth.acs')
   ('/user/saml2login', 'saml2auth.saml2login')
   ('/user/login', 'saml2auth.disable_default_login_register')
   ('/user/register', 'saml2auth.disable_default_login_register')
   ('/slo', 'saml2auth.slo')
   ('/api/util/organization/autocomplete', 'api.organization_autocomplete')
   ('/api/util/resource/format_autocomplete', 'api.format_autocomplete')
   ('/api/util/dataset/autocomplete', 'api.dataset_autocomplete')
   ('/api/util/group/autocomplete', 'api.group_autocomplete')
   ('/api/util/user/autocomplete', 'api.user_autocomplete')
   ('/api/util/tag/autocomplete', 'api.tag_autocomplete')
   ('/ckan-admin/reset_config', 'admin.reset_config')
   ('/ckan-admin/config', 'admin.config')
   ('/ckan-admin/trash', 'admin.trash')
   ('/dashboard/organizations', 'dashboard.organizations')
   ('/dashboard/datasets', 'dashboard.datasets')
   ('/dashboard/groups', 'dashboard.groups')
   ('/testing/primer', 'util.primer')
   ('/feeds/dataset.atom', 'feeds.general')
   ('/feeds/custom.atom', 'feeds.custom')
   ('/user/logged_out_redirect', 'user.logged_out_page')
   ('/user/generate_key', 'user.generate_apikey')
   ('/user/logged_out', 'user.logged_out')
   ('/user/logged_in', 'user.logged_in')
   ('/user/register', 'user.register')
   ('/util/redirect', 'util.internal_redirect')
   ('/user/_logout', 'user.logout')
   ('/user/login', 'user.login')
   ('/user/reset', 'user.request_reset')
   ('/user/edit', 'user.edit')
   ('/user/me', 'user.me')
   ('/ckan-admin/', 'admin.index')
   ('/about', 'home.about')
   ('/user/', 'user.index')
   ('/api/', 'api.get_api')
   ('/', 'home.index')
   ('/api/<int(min=1, max=2):ver>/util/organization/autocomplete', 'api.organization_autocomplete')
   ('/api/<int(min=1, max=2):ver>/util/resource/format_autocomplete', 'api.format_autocomplete')
   ('/api/<int(min=1, max=2):ver>/util/dataset/autocomplete', 'api.dataset_autocomplete')
   ('/api/<int(min=1, max=2):ver>/util/group/autocomplete', 'api.group_autocomplete')
   ('/api/<int(min=1, max=2):ver>/util/user/autocomplete', 'api.user_autocomplete')
   ('/api/<int(min=1, max=2):ver>/util/tag/autocomplete', 'api.tag_autocomplete')
   ('/dataset/<id>/resource/new', 'resource.new')
   ('/dataset/<id>/resource/new', 'dataset_resource.new')
   ('/harvest/<id>/resource/new', 'harvest_resource.new')
   ('/dataset/<id>/resource/<resource_id>/edit_view/<view_id>', 'resource.edit_view')
   ('/dataset/<id>/resource/<resource_id>/edit_view/<view_id>', 'dataset_resource.edit_view')
   ('/harvest/<id>/resource/<resource_id>/edit_view/<view_id>', 'harvest_resource.edit_view')
   ('/dataset/<id>/resource/<resource_id>/download/<filename>', 'resource.download')
   ('/dataset/<id>/resource/<resource_id>/download/<filename>', 'dataset_resource.download')
   ('/harvest/<id>/resource/<resource_id>/download/<filename>', 'harvest_resource.download')
   ('/dataset/<id>/resource/<resource_id>/download', 'resource.download')
   ('/dataset/<id>/resource/<resource_id>/new_view', 'resource.edit_view')
   ('/dataset/<id>/resource/<resource_id>/download', 'dataset_resource.download')
   ('/dataset/<id>/resource/<resource_id>/new_view', 'dataset_resource.edit_view')
   ('/harvest/<id>/resource/<resource_id>/download', 'harvest_resource.download')
   ('/harvest/<id>/resource/<resource_id>/new_view', 'harvest_resource.edit_view')
   ('/dataset/<id>/resource/<resource_id>/preview', 'resource.datapreview')
   ('/dataset/<id>/resource/<resource_id>/preview', 'dataset_resource.datapreview')
   ('/harvest/<id>/resource/<resource_id>/preview', 'harvest_resource.datapreview')
   ('/dataset/<id>/resource/<resource_id>/delete', 'resource.delete')
   ('/dataset/<id>/resource/<resource_id>/viewer', 'resource.embedded_dataviewer')
   ('/dataset/<id>/resource/<resource_id>/delete', 'dataset_resource.delete')
   ('/dataset/<id>/resource/<resource_id>/viewer', 'dataset_resource.embedded_dataviewer')
   ('/harvest/<id>/resource/<resource_id>/delete', 'harvest_resource.delete')
   ('/harvest/<id>/resource/<resource_id>/viewer', 'harvest_resource.embedded_dataviewer')
   ('/dataset/<id>/resource/<resource_id>/views', 'resource.views')
   ('/dataset/<id>/resource/<resource_id>/embed', 'resource.embedded_dataviewer')
   ('/dataset/<id>/resource/<resource_id>/views', 'dataset_resource.views')
   ('/dataset/<id>/resource/<resource_id>/embed', 'dataset_resource.embedded_dataviewer')
   ('/harvest/<id>/resource/<resource_id>/views', 'harvest_resource.views')
   ('/harvest/<id>/resource/<resource_id>/embed', 'harvest_resource.embedded_dataviewer')
   ('/dataset/<id>/resource/<resource_id>/view/<view_id>', 'resource.view')
   ('/dataset/<id>/resource/<resource_id>/view/<view_id>', 'dataset_resource.view')
   ('/harvest/<id>/resource/<resource_id>/view/<view_id>', 'harvest_resource.view')
   ('/dataset/<id>/resource/<resource_id>/edit', 'resource.edit')
   ('/dataset/<id>/resource/<resource_id>/view', 'resource.view')
   ('/dataset/<id>/resource/<resource_id>/edit', 'dataset_resource.edit')
   ('/dataset/<id>/resource/<resource_id>/view', 'dataset_resource.view')
   ('/harvest/<id>/resource/<resource_id>/edit', 'harvest_resource.edit')
   ('/harvest/<id>/resource/<resource_id>/view', 'harvest_resource.view')
   ('/feeds/organization/<string:id>.atom', 'feeds.organization')
   ('/feeds/group/<string:id>.atom', 'feeds.group')
   ('/feeds/tag/<string:id>.atom', 'feeds.tag')
   ('/user/<id>/api-tokens/<jti>/revoke', 'user.api_token_revoke')
   ('/api/util/snippet/<snippet_path>', 'api.snippet')
   ('/api/<int(min=1, max=2):ver>/util/snippet/<snippet_path>', 'api.snippet')
   ('/organization/member_delete/<id>', 'organization.member_delete')
   ('/organization/bulk_process/<id>', 'organization.bulk_process')
   ('/organization/member_new/<id>', 'organization.member_new')
   ('/organization/followers/<id>', 'organization.followers')
   ('/organization/activity/<id>/<int:offset>', 'organization.activity')
   ('/organization/unfollow/<id>', 'organization.unfollow')
   ('/organization/activity/<id>', 'organization.activity')
   ('/organization/members/<id>', 'organization.members')
   ('/organization/history/<id>', 'organization.history')
   ('/organization/delete/<id>', 'organization.delete')
   ('/organization/follow/<id>', 'organization.follow')
   ('/organization/admins/<id>', 'organization.admins')
   ('/organization/about/<id>', 'organization.about')
   ('/organization/edit/<id>', 'organization.edit')
   ('/organization/new', 'organization.new')
   ('/dataset/changes_multiple', 'dataset.changes_multiple')
   ('/harvest/changes_multiple', 'harvest.changes_multiple')
   ('/dataset/new_resource/<id>', 'dataset.CreateView')
   ('/harvest/new_resource/<id>', 'harvest.CreateView')
   ('/dataset/resources/<id>', 'dataset.resources')
   ('/dataset/followers/<id>', 'dataset.followers')
   ('/harvest/resources/<id>', 'harvest.resources')
   ('/harvest/followers/<id>', 'harvest.followers')
   ('/dataset/unfollow/<id>', 'dataset.unfollow')
   ('/dataset/activity/<id>', 'dataset.activity')
   ('/harvest/unfollow/<id>', 'harvest.unfollow')
   ('/harvest/activity/<id>', 'harvest.activity')
   ('/dataset/changes/<id>', 'dataset.changes')
   ('/harvest/changes/<id>', 'harvest.changes')
   ('/dataset/delete/<id>', 'dataset.delete')
   ('/dataset/follow/<id>', 'dataset.follow')
   ('/dataset/groups/<id>', 'dataset.groups')
   ('/harvest/delete/<id>', 'harvest.delete')
   ('/harvest/follow/<id>', 'harvest.follow')
   ('/harvest/groups/<id>', 'harvest.groups')
   ('/dataset/edit/<id>', 'dataset.edit')
   ('/harvest/edit/<id>', 'harvest.edit')
   ('/dataset/new', 'dataset.new')
   ('/harvest/new', 'harvest.new')
   ('/dataset/<id>/resource_edit/<resource_id>', 'dataset.EditView')
   ('/harvest/<id>/resource_edit/<resource_id>', 'harvest.EditView')
   ('/dataset/<id>/resource/<resource_id>', 'resource.read')
   ('/dataset/<id>/resource/<resource_id>', 'dataset_resource.read')
   ('/harvest/<id>/resource/<resource_id>', 'harvest_resource.read')
   ('/dataset/<id>/history', 'dataset.history')
   ('/harvest/<id>/history', 'harvest.history')
   ('/group/member_delete/<id>', 'group.member_delete')
   ('/group/bulk_process/<id>', 'group.bulk_process')
   ('/group/member_new/<id>', 'group.member_new')
   ('/group/followers/<id>', 'group.followers')
   ('/group/activity/<id>/<int:offset>', 'group.activity')
   ('/group/unfollow/<id>', 'group.unfollow')
   ('/group/activity/<id>', 'group.activity')
   ('/group/members/<id>', 'group.members')
   ('/group/history/<id>', 'group.history')
   ('/group/delete/<id>', 'group.delete')
   ('/group/follow/<id>', 'group.follow')
   ('/group/admins/<id>', 'group.admins')
   ('/group/about/<id>', 'group.about')
   ('/group/edit/<id>', 'group.edit')
   ('/group/new', 'group.new')
   ('/user/generate_key/<id>', 'user.generate_apikey')
   ('/user/followers/<id>', 'user.followers')
   ('/user/activity/<id>/<int:offset>', 'user.activity')
   ('/user/activity/<id>', 'user.activity')
   ('/user/unfollow/<id>', 'user.unfollow')
   ('/user/delete/<id>', 'user.delete')
   ('/user/follow/<id>', 'user.follow')
   ('/user/reset/<id>', 'user.perform_reset')
   ('/user/edit/<id>', 'user.edit')
   ('/user/<id>/api-tokens', 'user.api_tokens')
   ('/api/action/<logic_function>', 'api.action')
   ('/api/i18n/<lang>', 'api.i18n_js_translations')
   ('/api/<int(min=3, max=3):ver>/action/<logic_function>', 'api.action')
   ('/api/<int(min=1, max=2):ver>/i18n/<lang>', 'api.i18n_js_translations')
   ('/organization/<id>', 'organization.read')
   ('/organization/', 'organization.index')
   ('/dashboard/<int:offset>', 'dashboard.index')
   ('/webassets/<path:path>', 'webassets.index')
   ('/dashboard/', 'dashboard.index')
   ('/dataset/<id>', 'dataset.read')
   ('/harvest/<id>', 'harvest.read')
   ('/dataset/', 'dataset.search')
   ('/harvest/', 'harvest.search')
   ('/group/<id>', 'group.read')
   ('/zh_TW/<path:path>', 'home.redirect_locale')
   ('/zh_CN/<path:path>', 'home.redirect_locale')
   ('/group/', 'group.index')
   ('/zh_TW/', 'home.redirect_locale')
   ('/zh_CN/', 'home.redirect_locale')
   ('/user/<id>', 'user.read')
   ('/api/<int(min=1, max=3):ver>', 'api.get_api')
   ('/<path:filename>', 'static')
   ```
</details>

<details>
  <summary>Inventory routes</summary>

   ```python
   ('/datastore/dump/<resource_id>', 'datastore.dump')
   ('/dataset/<id>/dictionary/<resource_id>', 'datastore.dictionary')
   ('/dataset/<id>/resource_data/<resource_id>', 'xloader.resource_data')
   ('/stats', 'stats.index')
   ('/dataset/<id>/resource/<resource_id>/download', 's3_resource.resource_download')
   ('/dataset/<id>/resource/<resource_id>/download/<filename>', 's3_resource.resource_download')
   ('/dataset/<id>/resource/<resource_id>/fs_download/<filename>', 's3_resource.filesystem_resource_download')
   ('/uploads/<upload_to>/<filename>', 's3_uploads.uploaded_file_redirect')
   ('/dataset/edit-new/<id>', 'dcat_usmetadata.edit_metadata')
   ('/dataset/new-metadata', 'dcat_usmetadata.new_metadata')
   ('/dataset/new_resource/<id>', 'usmetadata.new_resource_usmetadata')
   ('/api/2/util/resource/license_url_autocomplete', 'usmetadata.lc_get_license_url')
   ('/dataset/<id>/clone', 'usmetadata.cc_clone_dataset_metadata')
   ('/api/2/util/resource/media_autocomplete', 'usmetadata.mc_get_media_types')
   ('/api/2/util/resource/content_type', 'usmetadata.cuc_get_content_type')
   ('/api/2/util/resource/validate_resource', 'usmetadata.rv_validate_resource')
   ('/api/2/util/resource/validate_dataset', 'usmetadata.dv_validate_dataset')
   ('/data.json', 'datajson.generate_json')
   ('/organization/<org_id>/data.json', 'datajson.generate_org_json')
   ('/organization/<org_id>/redacted.json', 'datajson.generate_redacted')
   ('/organization/<org_id>/unredacted.json', 'datajson.generate_unredacted')
   ('/organization/<org_id>/draft.json', 'datajson.generate_draft')
   ('/pod/validate', 'datajson.validator')
   ('/acs', 'saml2auth.acs')
   ('/user/saml2login', 'saml2auth.saml2login')
   ('/user/login', 'saml2auth.disable_default_login_register')
   ('/user/register', 'saml2auth.disable_default_login_register')
   ('/slo', 'saml2auth.slo')
   ('/api/util/organization/autocomplete', 'api.organization_autocomplete')
   ('/api/util/resource/format_autocomplete', 'api.format_autocomplete')
   ('/api/util/dataset/autocomplete', 'api.dataset_autocomplete')
   ('/api/util/group/autocomplete', 'api.group_autocomplete')
   ('/api/util/user/autocomplete', 'api.user_autocomplete')
   ('/api/util/tag/autocomplete', 'api.tag_autocomplete')
   ('/ckan-admin/reset_config', 'admin.reset_config')
   ('/ckan-admin/config', 'admin.config')
   ('/ckan-admin/trash', 'admin.trash')
   ('/dashboard/organizations', 'dashboard.organizations')
   ('/dashboard/datasets', 'dashboard.datasets')
   ('/dashboard/groups', 'dashboard.groups')
   ('/testing/primer', 'util.primer')
   ('/feeds/dataset.atom', 'feeds.general')
   ('/feeds/custom.atom', 'feeds.custom')
   ('/user/logged_out_redirect', 'user.logged_out_page')
   ('/user/generate_key', 'user.generate_apikey')
   ('/user/logged_out', 'user.logged_out')
   ('/user/logged_in', 'user.logged_in')
   ('/user/register', 'user.register')
   ('/util/redirect', 'util.internal_redirect')
   ('/user/_logout', 'user.logout')
   ('/user/login', 'user.login')
   ('/user/reset', 'user.request_reset')
   ('/user/edit', 'user.edit')
   ('/user/me', 'user.me')
   ('/ckan-admin/', 'admin.index')
   ('/about', 'home.about')
   ('/user/', 'user.index')
   ('/api/', 'api.get_api')
   ('/', 'home.index')
   ('/api/<int(min=1, max=2):ver>/util/organization/autocomplete', 'api.organization_autocomplete')
   ('/api/<int(min=1, max=2):ver>/util/resource/format_autocomplete', 'api.format_autocomplete')
   ('/api/<int(min=1, max=2):ver>/util/dataset/autocomplete', 'api.dataset_autocomplete')
   ('/api/<int(min=1, max=2):ver>/util/group/autocomplete', 'api.group_autocomplete')
   ('/api/<int(min=1, max=2):ver>/util/user/autocomplete', 'api.user_autocomplete')
   ('/api/<int(min=1, max=2):ver>/util/tag/autocomplete', 'api.tag_autocomplete')
   ('/dataset/<id>/resource/new', 'resource.new')
   ('/dataset/<id>/resource/new', 'dataset_resource.new')
   ('/dataset/<id>/resource/<resource_id>/edit_view/<view_id>', 'resource.edit_view')
   ('/dataset/<id>/resource/<resource_id>/edit_view/<view_id>', 'dataset_resource.edit_view')
   ('/dataset/<id>/resource/<resource_id>/download/<filename>', 'resource.download')
   ('/dataset/<id>/resource/<resource_id>/download/<filename>', 'dataset_resource.download')
   ('/dataset/<id>/resource/<resource_id>/download', 'resource.download')
   ('/dataset/<id>/resource/<resource_id>/new_view', 'resource.edit_view')
   ('/dataset/<id>/resource/<resource_id>/download', 'dataset_resource.download')
   ('/dataset/<id>/resource/<resource_id>/new_view', 'dataset_resource.edit_view')
   ('/dataset/<id>/resource/<resource_id>/preview', 'resource.datapreview')
   ('/dataset/<id>/resource/<resource_id>/preview', 'dataset_resource.datapreview')
   ('/dataset/<id>/resource/<resource_id>/delete', 'resource.delete')
   ('/dataset/<id>/resource/<resource_id>/viewer', 'resource.embedded_dataviewer')
   ('/dataset/<id>/resource/<resource_id>/delete', 'dataset_resource.delete')
   ('/dataset/<id>/resource/<resource_id>/viewer', 'dataset_resource.embedded_dataviewer')
   ('/dataset/<id>/resource/<resource_id>/views', 'resource.views')
   ('/dataset/<id>/resource/<resource_id>/embed', 'resource.embedded_dataviewer')
   ('/dataset/<id>/resource/<resource_id>/views', 'dataset_resource.views')
   ('/dataset/<id>/resource/<resource_id>/embed', 'dataset_resource.embedded_dataviewer')
   ('/dataset/<id>/resource/<resource_id>/view/<view_id>', 'resource.view')
   ('/dataset/<id>/resource/<resource_id>/view/<view_id>', 'dataset_resource.view')
   ('/dataset/<id>/resource/<resource_id>/edit', 'resource.edit')
   ('/dataset/<id>/resource/<resource_id>/view', 'resource.view')
   ('/dataset/<id>/resource/<resource_id>/edit', 'dataset_resource.edit')
   ('/dataset/<id>/resource/<resource_id>/view', 'dataset_resource.view')
   ('/feeds/organization/<string:id>.atom', 'feeds.organization')
   ('/feeds/group/<string:id>.atom', 'feeds.group')
   ('/feeds/tag/<string:id>.atom', 'feeds.tag')
   ('/user/<id>/api-tokens/<jti>/revoke', 'user.api_token_revoke')
   ('/api/util/snippet/<snippet_path>', 'api.snippet')
   ('/api/<int(min=1, max=2):ver>/util/snippet/<snippet_path>', 'api.snippet')
   ('/organization/member_delete/<id>', 'organization.member_delete')
   ('/organization/bulk_process/<id>', 'organization.bulk_process')
   ('/organization/member_new/<id>', 'organization.member_new')
   ('/organization/followers/<id>', 'organization.followers')
   ('/organization/activity/<id>/<int:offset>', 'organization.activity')
   ('/organization/unfollow/<id>', 'organization.unfollow')
   ('/organization/activity/<id>', 'organization.activity')
   ('/organization/members/<id>', 'organization.members')
   ('/organization/history/<id>', 'organization.history')
   ('/organization/delete/<id>', 'organization.delete')
   ('/organization/follow/<id>', 'organization.follow')
   ('/organization/admins/<id>', 'organization.admins')
   ('/organization/about/<id>', 'organization.about')
   ('/organization/edit/<id>', 'organization.edit')
   ('/organization/new', 'organization.new')
   ('/dataset/changes_multiple', 'dataset.changes_multiple')
   ('/dataset/new_resource/<id>', 'dataset.CreateView')
   ('/dataset/resources/<id>', 'dataset.resources')
   ('/dataset/followers/<id>', 'dataset.followers')
   ('/dataset/unfollow/<id>', 'dataset.unfollow')
   ('/dataset/activity/<id>', 'dataset.activity')
   ('/dataset/changes/<id>', 'dataset.changes')
   ('/dataset/delete/<id>', 'dataset.delete')
   ('/dataset/follow/<id>', 'dataset.follow')
   ('/dataset/groups/<id>', 'dataset.groups')
   ('/dataset/edit/<id>', 'dataset.edit')
   ('/dataset/new', 'dataset.new')
   ('/dataset/<id>/resource_edit/<resource_id>', 'dataset.EditView')
   ('/dataset/<id>/resource/<resource_id>', 'resource.read')
   ('/dataset/<id>/resource/<resource_id>', 'dataset_resource.read')
   ('/dataset/<id>/history', 'dataset.history')
   ('/group/member_delete/<id>', 'group.member_delete')
   ('/group/bulk_process/<id>', 'group.bulk_process')
   ('/group/member_new/<id>', 'group.member_new')
   ('/group/followers/<id>', 'group.followers')
   ('/group/activity/<id>/<int:offset>', 'group.activity')
   ('/group/unfollow/<id>', 'group.unfollow')
   ('/group/activity/<id>', 'group.activity')
   ('/group/members/<id>', 'group.members')
   ('/group/history/<id>', 'group.history')
   ('/group/delete/<id>', 'group.delete')
   ('/group/follow/<id>', 'group.follow')
   ('/group/admins/<id>', 'group.admins')
   ('/group/about/<id>', 'group.about')
   ('/group/edit/<id>', 'group.edit')
   ('/group/new', 'group.new')
   ('/user/generate_key/<id>', 'user.generate_apikey')
   ('/user/followers/<id>', 'user.followers')
   ('/user/activity/<id>/<int:offset>', 'user.activity')
   ('/user/activity/<id>', 'user.activity')
   ('/user/unfollow/<id>', 'user.unfollow')
   ('/user/delete/<id>', 'user.delete')
   ('/user/follow/<id>', 'user.follow')
   ('/user/reset/<id>', 'user.perform_reset')
   ('/user/edit/<id>', 'user.edit')
   ('/user/<id>/api-tokens', 'user.api_tokens')
   ('/api/action/<logic_function>', 'api.action')
   ('/api/i18n/<lang>', 'api.i18n_js_translations')
   ('/api/<int(min=3, max=3):ver>/action/<logic_function>', 'api.action')
   ('/api/<int(min=1, max=2):ver>/i18n/<lang>', 'api.i18n_js_translations')
   ('/organization/<id>', 'organization.read')
   ('/organization/', 'organization.index')
   ('/dashboard/<int:offset>', 'dashboard.index')
   ('/webassets/<path:path>', 'webassets.index')
   ('/dashboard/', 'dashboard.index')
   ('/dataset/<id>', 'dataset.read')
   ('/dataset/', 'dataset.search')
   ('/group/<id>', 'group.read')
   ('/zh_TW/<path:path>', 'home.redirect_locale')
   ('/zh_CN/<path:path>', 'home.redirect_locale')
   ('/group/', 'group.index')
   ('/zh_TW/', 'home.redirect_locale')
   ('/zh_CN/', 'home.redirect_locale')
   ('/user/<id>', 'user.read')
   ('/api/<int(min=1, max=3):ver>', 'api.get_api')
   ('/<path:filename>', 'static')

   ```
</details>

**Comment by nickumia-reisys** [comment]
<details>
  <summary>Parent Inventory routes</summary>

   ```bash
'/stats'
'/data.json'
'/organization'
'/feeds'
'/ckan-admin'
'/about'
'/api'
'/'
'/group'
'/webassets'
'/dashboard'
'/user'
'/fonts'
'/base/vendor'
('/<path:filename>', 'static')
('/acs', 'saml2auth.acs')
('/slo', 'saml2auth.slo')
('/datastore/dump/<resource_id>', 'datastore.dump')
('/dataset/<id>/dictionary/<resource_id>', 'datastore.dictionary')
('/uploads/<upload_to>/<filename>', 's3_uploads.uploaded_file_redirect')
('/testing/primer', 'util.primer')
```
</details>

**Comment by nickumia-reisys** [comment]
The only things not explicitly captured in the above lists of routes are statically host files that exist outside of the `webassets` subdir:
- `robots.txt`
- any special fonts and things
  - Example: https://github.com/GSA/ckanext-datagovtheme/pull/141

**Comment by hkdctol** [comment]
This can be closed in light of #4044
